source/_posts/2015-09-03-hiring-apprentices.markdown

```diff
@@ -0,0 +1,82 @@
+I also love my job at MX so I liked the idea of helping to grow our team.
+
+When I discussed the idea of an apprenticeship with people at MX, the first thing I had to do was nail down what our objective was. "Helping new engineers" wasn't going to cut it.
+
+We decided that the primary objective was to __add amazing engineers to the team__.
+
+# Amazing Apprentices
+
+Based on my experience in the local Ruby community I believed that there were some really amazing programmers who were self-taught or graduating from bootcamps that were struggling to find first jobs.
+
+The shortage of senior engineers at pretty much every company has made it really difficult to get a job as a junior developer. Lots of companies think that they can't afford to hire junior engineers because they don't have enough senior people to train them.
+There are also serious bias issues throughout the software industry and especially in the hiring processes.
+
+So we had a hypothesis that there was a large pool of extremely talented and hard-working people that was underutilized.
+
+# Teaching
+
+On our team we usually only considered hiring someone if they had a full year of professional programming experience.
+This is partially because our team has no managers and no task assignment so it is easy to get lost in the weeds if you don't have your own discipline for staying focused as an engineer.
+But this meant that if we wanted to make an apprenticeship work for us, we needed  the apprentices to have a skill and experience level similar to someone with one full year of professional experience.
+
+But I didn't think it should actually take a year to gain that experience and those skills.
+
+It is all too common for people to have a first programming job where they get assigned a laundry list of bugs and work alone [spending lots of time](http://www.happybearsoftware.com/how-to-survive-your-first-year-as-a-programmer.html) on just a few small conceptual misunderstandings.
+In my own experience my first two jobs wasted a lot of time because the experienced people on the team were too busy to provide guidance on which things were worth learning and which things I should skip until later.
+I rarely got feedback on my projects unless they broke something in production.
+
+I'm not qualified to write a decent curriculum, but I think that we can deliberately make a lot of time for smart people to talk through what they are learning.
+I think that with this kind of feedback the apprentices will significantly outpace the rate at which I learned things as a new developer.
+I also think that by hiring apprentices in a group and giving them tasks as a group, they will do a lot of learning as a team.
+This will help prepare them for the very chaotic and collaborative environment on the MX engineering team, and also speed up their learning because they spend time teaching and learning from people with a similar experience level.
```

Comment:
"and also" is redundant. Simply "and" works just fine. Also, repeated words are repeated. Too much "learning."